node/src/components/consensus/highway_core/evidence.rs

```diff
@@ -30,13 +44,29 @@ pub(crate) enum EvidenceError {
 pub(crate) enum Evidence<C: Context> {
     /// The validator produced two units with the same sequence number.
     Equivocation(SignedWireUnit<C>, SignedWireUnit<C>),
+    /// The validator endorsed two conflicting units.
+    Endorsements {
+        /// The endorsement for `unit1`.
+        endorsement1: SignedEndorsement<C>,
+        /// The unit with the lower (or equal) sequence number.
+        unit1: SignedWireUnit<C>,
+        /// The endorsement for `unit2`, by the same creator as endorsement1.
+        endorsement2: SignedEndorsement<C>,
+        /// The unit with the higher sequence number, on a conflicting fork of the same creator as
+        /// `unit1`.
```

Comment:
Why does it have to have higher sequence number, why it can't be equal?

Reply:
It can! I'll update the comment.